frontend/public/locales/en/persistent-volume-claim.json

```diff
@@ -1,6 +1,8 @@
 {
   "PersistentVolumeClaim details": "PersistentVolumeClaim details",
   "Label selector": "Label selector",
+  "Selector": "Selector",
+  "Add Value": "Add Value",
```

Comment:
I'm not sure where should they go~`public.json` or `persistent-volume-claim.json`. How do we determine where should they reside?

Reply:
Hi, @yapei. We're moving to having a namespace per package in the console instead of a separate namespace for each module. So I would just put this in `public`. `persistent-volume-claim.json` will eventually go away.

Reply:
thank you @spadgett  updated